core/dblookupext/historyRepository_test.go

```diff
@@ -594,6 +596,31 @@ func TestHistoryRepository_CommitOnForkThenNewEpochThenCommit(t *testing.T) {
 	require.Nil(t, orphanedMetadata.NotarizedAtDestinationInMetaHash)
 }
 
+func TestHistoryRepository_getMiniblockMetadataByMiniblockHashGetFromEpochErrorsShouldErr(t *testing.T) {
```

Comment:
why this test is needed was a problem with coverage here?

Reply:
this test covers an area that is only sometimes reached by the next suit of tests. I could not made those tests work more deterministic as it uses 2 go routines that concurrently call some functions. (tried synchronizing the go routines, locally the test passes each time, on github action seldom fails)

Reply:
Now I understand your reason. Thank you for your clarification.